apps/meteor/tests/e2e/omnichannel/omnichannel-units.spec.ts

```diff
@@ -117,6 +122,24 @@ test.describe('OC - Manage Units', () => {
 			await expect(poOmnichannelUnits.findRowByName(editedUnitName)).toBeVisible();
 		});
 
+		await test.step('expect to add another monitor to list', async () => {
+			await poOmnichannelUnits.findRowByName(editedUnitName).click();
+			await expect(poOmnichannelUnits.contextualBar).toBeVisible();
+			await poOmnichannelUnits.selectMonitor('user3');
+			await poOmnichannelUnits.btnSave.click();
+		});
+
+		await test.step('expect unit to have been edited with 2 monitors', async () => {
+			await expect(poOmnichannelUnits.inputSearch).toBeVisible();
+			await poOmnichannelUnits.search(editedUnitName);
+			await expect(poOmnichannelUnits.findRowByName(editedUnitName)).toBeVisible();
+			await poOmnichannelUnits.findRowByName(editedUnitName).click();
+			await expect(poOmnichannelUnits.contextualBar).toBeVisible();
+
+			await expect(poOmnichannelUnits.inputMonitors).toHaveText(/user2/);
+			await expect(poOmnichannelUnits.inputMonitors).toHaveText(/user3/);
+		});
+
```

Comment:
Can we add a new test case removing one user from the list (of 2), save it, and see if it behaves correctly?